src/olympia/scanners/admin.py

```diff
@@ -178,6 +179,20 @@ def has_delete_permission(self, request, obj=None):
     def has_change_permission(self, request, obj=None):
         return False
 
+    def get_list_display(self, request):
+        fields = super().get_list_display(request)
+        return self._excludes_admin_fields(request=request, fields=fields)
+
+    def get_fields(self, request, obj=None):
```

Comment:
I see tests for `test_list_view`, but I'm not sure if those are testing `get_fields` or `get_list_display` (or both, although that seems unlikely). I guess my question is: do we have added coverage for both of these methods?

Reply:
We don't for this one specifically, but both use the same (private) method, which is tested _via_ the `test_list_diplay()` and `test_list_display_for_non_admins()`.